app/src/main/res/values/strings.xml

```diff
@@ -15,4 +15,6 @@
 
     <!-- TODO: Remove or change this placeholder text -->
     <string name="hello_blank_fragment">Hello blank fragment</string>
+    <string name="searchable">search</string>
```

Comment:
I think a better name for this will be `query_text` or something more descriptive. Good that you put all the strings in the `strings.xml` file